tests/core/data/test_parquet_data_node.py

```diff
@@ -252,20 +252,25 @@ def test_get_system_file_modified_date_instead_of_last_edit_date(self, tmpdir_fa
     def test_get_system_folder_modified_date_instead_of_last_edit_date(self, tmpdir_factory):
         temp_folder_path = tmpdir_factory.mktemp("data").strpath
         temp_file_path = os.path.join(temp_folder_path, "temp.parquet")
-        dn = ParquetDataNode("foo", Scope.PIPELINE, properties={"path": temp_folder_path, "exposed_type": "pandas"})
-
         pd.DataFrame([]).to_parquet(temp_file_path)
 
-        assert dn.last_edit_date == datetime.fromtimestamp(os.path.getmtime(temp_file_path))
-        previous_edit_date = dn.last_edit_date
+        dn = ParquetDataNode("foo", Scope.PIPELINE, properties={"path": temp_folder_path})
+        initial_edit_date = dn.last_edit_date
+
+        sleep(0.1)
```

Comment:
Can you let a comment to explain why?

Reply:
I did edited the above comment. But I guess here is much better 😅

So the problem seems like the after the creation of a file, Ubuntu does something with the metadata of the file, which after a few millisec change the modified date of the file.
This messes up the test since we instantly get the modified time of the file after creation.

To fix this, the file is created first in the test. Then the test was carried-on as usual.

Reply:
I was talking about a comment in the code for the following contributors... so they know why we sleep.

🤣 🤣

Reply:
Ah ... my bad

Reply:
YEp. It s clear enough.